README.md

```diff
@@ -28,21 +30,64 @@ nvm use
 yarn
 ```
 
-### 3. Run MongoDB
+### 3. Configure environment
 
-Prerequisites:
+#### `/.env.template`
 
-- Docker installed
+Copy and rename `.env`.
+
+Set server domain:
+
+```
+SERVER=localhost
+```
+
+Configure database usernames and passwords:
 
-Copy root `.env.template` and rename `.env`. Configure database usernames and passwords:
+```
+MONGO_DB=praise_db
+MONGO_HOST=mongodb
+MONGO_PORT=27017
+MONGO_INITDB_ROOT_USERNAME=praiseDbRootUsername
+MONGO_INITDB_ROOT_PASSWORD=[any password]
+MONGO_USERNAME=praiseDbUsername
+MONGO_PASSWORD=[any password]
+```
+
+#### `/packages/api/.env.template`
+
+Copy and rename `.env`.
+
+Add your Metamask ETH address to `ADMINS` to be able to access Praise dashboard as admin:
 
 ```
-MONGO_INITDB_ROOT_USERNAME=
-MONGO_INITDB_ROOT_PASSWORD=
-MONGO_DB_USER=
-MONGO_DB_PASSWORD=
+ADMINS=0x123..123,0x123..12
 ```
 
+#### `/packages/discord-bot/.env.template`
+
+Copy and rename `.env`.
+
+- `DISCORD_TOKEN` - Your bot's discord token generated via the Discord Developer Portal. You'll need to invite the same bot to your server with the link - `https://discord.com/api/oauth2/authorize?client_id=<client-id>&permissions=378561611840&scope=bot%20applications.commands` (replace `<client-id>` with your bot's client ID), and with the SERVER MEMBERS and MESSAGE CONTENT Intents enabled.
+- `DISCORD_CLIENT_ID` - Your bot's discord client ID, which can be found in the Application settings on Discord Developer Portal
+- `DISCORD_GUILD_ID` - The ID of the server in which you are using the bot. (this can be found by enabling developer mode in Discord, right clicking on te server icon and clicking "Copy Id").
+
+```
+DISCORD_TOKEN=
+DISCORD_CLIENT_ID=
+DISCORD_GUILD_ID=
+```
+
+#### `/packages/frontend/.env.template`
+
+Copy and rename `.env.development`.
```

Comment:
Any reason not to just copy the env file to `.env`? Rather than `.env.development`?

Reply:
I think current setup of React requires `.env.development` or `.env.production`. I would have preferred removing frontend env completely but React needs separate variables beginning with `REACT_APP`.

Reply:
It's kinda confusing but it looks like with react-scripts `.env` is always the "default" config, with others overriding it conditionally:
https://create-react-app.dev/docs/adding-custom-environment-variables/#what-other-env-files-can-be-used

Reply:
Ah, nice, didn't know that! Will change then.